packages/cli/src/tools/assign.ts

```diff
@@ -4,14 +4,19 @@
  * Similar to Object.assign(), but it doesn't execute getters. This allows us to have
  * lazy properties on an object and still be able to merge them together
  *
- * @flow
  */
 export default function assign(target: Object, ...sources: Object[]) {
   sources.forEach(source => {
-    let descriptors = Object.keys(source).reduce((acc, key) => {
-      acc[key] = Object.getOwnPropertyDescriptor(source, key);
-      return acc;
-    }, {});
+    let descriptors = Object.keys(source).reduce(
+      (acc, key) => {
+        const propertyDescriptor = Object.getOwnPropertyDescriptor(source, key);
+        if (propertyDescriptor !== undefined) {
+          acc[key] = propertyDescriptor;
+        }
+        return acc;
+      },
+      {} as PropertyDescriptorMap,
```

Comment:
I guess it's more readable with the typing in the parameters?!?

```typescript
(acc: PropertyDescriptorMap, key: string) => {
```

So you avoid the typecasting in the initial value ```as PropertyDescriptorMap ```

But it's just my opnion 🤔

Reply:
IMO it's a bit also more readable, but we can wait for @thymikee opinion

Reply:
I prefer `{} as PropertyDescriptorMap`. As long as TS is able to give me correct autocompletion, it's better to rely on inference and type less imho :)